### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by TimGebert**
Added PR Template
Added Contributing document
Updated API Descriptions

### CODE REVIEW

File: packages/server/src/db/plan-model.tsx [hunk 1]
```diff
@@ -63,11 +63,36 @@ export function createPlanService(planModel) {
...
+            type: 'string',
+            description: 'Language that the plan will generate an SDK for',
+          },
+          version: {
+            type: 'string',
+            description: 'The version number of the SDK',
+          },
+          options: {
+            type: 'object',
+            description: 'Additional options for the plan',
```
Comment (calebj0seph) [inline packages/server/src/db/plan-model.tsx]: This should be more descriptive, like 'The options to pass to OpenAPI Generator when generating the SDK'

File: packages/server/src/db/plan-model.tsx [hunk 2]
```diff
@@ -63,11 +63,36 @@ export function createPlanService(planModel) {
...
+            additionalProperties: true,
+          },
+          gitinfo: {
```
Comment (calebj0seph) [inline packages/server/src/db/plan-model.tsx]: This should be 'gitInfo'

File: packages/server/src/db/plan-model.tsx [hunk 3]
```diff
@@ -63,11 +63,36 @@ export function createPlanService(planModel) {
...
+            type: 'object',
```
Comment (calebj0seph) [inline packages/server/src/db/plan-model.tsx]: Can you add a definition for `GitInfo` and update this type? See [here](https://github.com/telstra/openapi-platform/blob/development/packages/model/src/GitInfo.tsx) for what the object contains. You'll also need to add a `GitAuth` type with parameters `username`, `password`, `token` and `oauth2format`. Those are described [here](https://isomorphic-git.org/docs/en/authentication.html).

File: packages/server/src/db/sdk-model.tsx [hunk 4]
```diff
@@ -37,7 +37,7 @@ export function createSdkService(sdkModel) {
     Model: sdkModel,
   });
   service.docs = {
-    description: 'The SDKs generated by Swagger/OpenAPI specs',
+    description: 'The SDKs generated by OpenAPI specs',
```
Comment (calebj0seph) [inline packages/server/src/db/sdk-model.tsx]: Maybe 'The SDKs generated from OpenAPI specifications' instead.

File: packages/server/src/db/sdk-model.tsx [hunk 5]
```diff
@@ -47,11 +47,18 @@ export function createSdkService(sdkModel) {
             format: 'int64',
             description: 'ID of the plan to generate the SDK for',
           },
+          path: {
+            type: 'string',
+            description: 'Url from where the SDK can be downloaded from',
```
Comment (calebj0seph) [inline packages/server/src/db/sdk-model.tsx]: Should be 'URL'.

File: packages/server/src/db/spec-model.tsx [hunk 6]
```diff
@@ -43,12 +43,29 @@ export function createSpecService(specModel) {
   service.docs = {
     description: 'Swagger/OpenAPI specs',
```
Comment (calebj0seph) [inline packages/server/src/db/spec-model.tsx]: Maybe 'The OpenAPI specifications to generate SDKs from' instead.

File: packages/server/src/db/spec-model.tsx [hunk 7]
```diff
@@ -43,12 +43,29 @@ export function createSpecService(specModel) {
...
+            type: 'string',
+            description: 'Name of the OpenAPI specification',
+          },
+          description: {
+            type: 'string',
+            description: 'Description of the OpenAPI specification',
+          },
+          path: {
+            type: 'string',
+            description: 'Url at which the OpenAPI specification can be found',
```
Comment (calebj0seph) [inline packages/server/src/db/spec-model.tsx]: Should be 'URL'.

File: packages/server/src/db/plan-model.tsx (hunk 3 above)
Comment (calebj0seph) [reply packages/server/src/db/plan-model.tsx]: If you wanna do this in a separate PR later that's fine, just fix the `gitInfo` thing and make note of this in Trello.

File: packages/server/src/db/plan-model.tsx (hunk 2 above)
Comment (PatrickShaw) [reply packages/server/src/db/plan-model.tsx]: Is this even added to the `sequelize` config yet?

**Pull request merged**